execution/model/operator.go

```diff
@@ -5,10 +5,63 @@ package model
 
 import (
 	"context"
+	"time"
 
 	"github.com/prometheus/prometheus/model/labels"
 )
 
+type NoopTelemetry struct {
+	name string
```

Comment:
Do we need this for noop? `SetName` can just be blank function, and `Name` can return empty string. :)